src/desktop/native/preload/Electron.js

```diff
@@ -477,18 +477,18 @@ const Electron = {
     exportState: (content) => {
         const prefix = 'Trinity';
 
-        const path = remote.dialog.showSaveDialog(remote.getCurrentWindow(), {
+        const result = remote.dialog.showSaveDialog(remote.getCurrentWindow(), {
```

Comment:
`remote.dialog.showSaveDialog` returns a Promise. We need to wait for the promise to get resolved before assigning it to result otherwise it will error out.

Reply:
Ah good catch, not sure how I missed that

Reply:
Resolved in 723eda6b6